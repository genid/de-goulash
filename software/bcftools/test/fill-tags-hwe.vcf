##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=1,assembly=b37,length=249250621>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	S1	S2	S3	S4	S5	S6	S7	S8	S9	S10
1	3177144	.	G	T	45	PASS	.	GT	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T	45	PASS	.	GT	1/0	1/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T	45	PASS	.	GT	1/1	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T,C	45	PASS	.	GT	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0
