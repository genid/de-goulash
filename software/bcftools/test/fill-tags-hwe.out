##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=1,assembly=b37,length=249250621>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes">
##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of samples with data">
##INFO=<ID=AC_Hom,Number=A,Type=Integer,Description="Allele counts in homozygous genotypes">
##INFO=<ID=AC_Het,Number=A,Type=Integer,Description="Allele counts in heterozygous genotypes">
##INFO=<ID=AC_Hemi,Number=A,Type=Integer,Description="Allele counts in hemizygous genotypes">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele frequency">
##INFO=<ID=MAF,Number=1,Type=Float,Description="Frequency of the second most common allele">
##INFO=<ID=HWE,Number=A,Type=Float,Description="HWE test (PMID:15789306); 1=good, 0=bad">
##INFO=<ID=ExcHet,Number=A,Type=Float,Description="Test excess heterozygosity; 1=good, 0=bad">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	S1	S2	S3	S4	S5	S6	S7	S8	S9	S10
1	3177144	.	G	T	45	PASS	NS=10;AN=20;AF=0;MAF=0;AC=0;AC_Het=0;AC_Hom=0;AC_Hemi=0;HWE=1;ExcHet=1	GT	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T	45	PASS	NS=10;AN=20;AF=0.1;MAF=0.1;AC=2;AC_Het=2;AC_Hom=0;AC_Hemi=0;HWE=1;ExcHet=0.947368	GT	1/0	1/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T	45	PASS	NS=10;AN=20;AF=0.1;MAF=0.1;AC=2;AC_Het=0;AC_Hom=2;AC_Hemi=0;HWE=0.0526316;ExcHet=1	GT	1/1	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0	0/0
1	3177144	.	G	T,C	45	PASS	NS=10;AN=20;AF=0.5,0;MAF=0.5;AC=10,0;AC_Het=10,0;AC_Hom=0,0;AC_Hemi=0,0;HWE=0.00690641,1;ExcHet=0.00554244,1	GT	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0	1/0
